app/src/main/res/values/styles.xml

```diff
@@ -614,6 +614,14 @@
         <item name="behavior_halfExpandedRatio">0.4</item>
     </style>
 
+    <style name="Mozac.Widgets.TopSite.Favicon" parent="Mozac.Widgets.Favicon">
```

Comment:
IMO mozac is still kept in resources names only when we intend to overwrite the value from AC.
Here we extend an AC style, but the above need does not exist.
Could we make it our own style with a simpler name, without "mozac"? Thoughts?

Reply:
Agreed with the above. I think it probably makes sense to drop the Mozac to something simpler in Fenix, and only keep to Mozac naming within AC.

Reply:
I think this is resolved.